apps/web/src/utils/chains.ts

```diff
@@ -41,6 +41,8 @@ export enum FEATURES {
   BRIDGE = 'BRIDGE',
   RENEW_NOTIFICATIONS_TOKEN = 'RENEW_NOTIFICATIONS_TOKEN',
   TX_NOTES = 'TX_NOTES',
+  NESTED_SAFES = 'NESTED_SAFES',
+  MASS_PAYOUTS = 'MASS_PAYOUTS',
```

Comment:
Should we prefix this with `TARGETED_` for clarity? Otherwise someone might toggle them in the config and expect the features to be enabled.

Reply:
Renamed in 927d53384247b970800270b9d19992306ea79470.